features/src/autogluon/features/generators/datetime.py

```diff
@@ -1,7 +1,8 @@
 import logging
-
 import pandas as pd
 from pandas import DataFrame
+from datetime import date
+import holidays
```

Comment:
unit test error: No module named holidays

Reply:
I added holidays library in setup.py